exp/constraints.lisp

```diff
@@ -102,6 +102,10 @@
                                (* PRPRC (next MACRO)))))
 
 ;; 6
+(defconstraint   wcp-flag-within-preprocessing ()
+                 (if-zero PRPRC
+                     (vanishes! preprocessing/WCP_FLAG)))
+;; 7
```

Comment:
Once the spec PR has been updated this constraint should be moved to the appropriate section.

Reply:
So this part should be removed.